elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java

```diff
@@ -1220,6 +1221,7 @@ public Resource toResource(final Supplier<Map<String, Relationship>> relationshi
                 : dictionary.getId(obj));
         resource.setRelationships(relationshipSupplier.get());
         resource.setAttributes(attributeSupplier.get());
+        resource.setLinks(getResourceLevelLink());
```

Comment:
(I feel) This should be configurable on a bunch of levels. You should be able to enable/disable links

* when creating the Elide instance
* when annotating a particular resource or relationship
* as a query param

I think a tri-state would make sense too. (`ForbidLinks`, `PermitLinks`, `ForceLinks`)